lib/livebook/config.ex

```diff
@@ -28,13 +28,35 @@ defmodule Livebook.Config do
   """
   @spec root_path() :: binary()
   def root_path() do
-    Application.get_env(:livebook, :root_path, File.cwd!())
+    Application.fetch_env!(:livebook, :root_path)
   end
 
   ## Parsing
 
   @doc """
-  Parses a long secret.
+  Sets the root path from env.
```

Comment:
I think `Sets` it's a bit confusing for these functions as they don't really mutate anything.